wallet-webapps-reward/src/main/webapp/vue-app/components/reward/SendRewardsTab.vue

```diff
@@ -305,6 +307,8 @@ export default {
   data() {
     return {
       search: '',
+      id: `DatePicker${parseInt(Math.random() * 10000)}`,
```

Comment:
could you use a timestampt instead a random here ?
In addition should id and menuId have the same timestamp or not ?

Reply:
> could you use a timestampt instead a random here ?
> In addition should id and menuId have the same timestamp or not ?

Done! It's not a problem I think when they have the same timestamp, the prefix is different ( DatePicker and DatePickerMenu)